contracts/strategies/uniswap/UniswapV3ConverterStrategy.sol

```diff
@@ -240,12 +222,11 @@ contract UniswapV3ConverterStrategy is UniswapV3Depositor, ConverterStrategyBase
         uint[] memory tokenAmounts = new uint[](2);
         tokenAmounts[0] = amount_;
         emit OnDepositorEnter(tokenAmounts, tokenAmounts);
-        _updateBaseAmountsForAsset(state.tokenA, amount_, 0);
       } else {
         (address[] memory tokens, uint indexAsset) = _getTokens(asset);
 
         // prepare array of amounts ready to deposit, borrow missed amounts
-        (uint[] memory amounts, uint[] memory borrowedAmounts, uint collateral) = _beforeDeposit(
+        (uint[] memory amounts,,) = _beforeDeposit(
```

Comment:
borrowedAmounts and collateral are not used anymore, probably _beforeDeposit (and getTokenAmounts) shouldn't return that values to reduce gas consumption.